internal/pkg/warewulfd/daemon.go

```diff
@@ -86,26 +88,22 @@ func DaemonStatus() error {
 
 func DaemonReload() error {
 	if !util.IsFile(WAREWULFD_PIDFILE) {
-		wwlog.Printf(wwlog.INFO, "Warewulf daemon process not running (%s)\n", WAREWULFD_PIDFILE)
-		return nil
+		return errors.New("Warewulf server is not running")
 	}
 
 	dat, err := ioutil.ReadFile(WAREWULFD_PIDFILE)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "could not read Warewulfd PID file")
 	}
 
 	pid, _ := strconv.Atoi(string(dat))
 	process, err := os.FindProcess(pid)
-
 	if err != nil {
-		fmt.Printf("Failed to find process: %s\n", err)
-		return err
+		return errors.Wrap(err, "failed to find running PID")
 	} else {
```

Comment:
No need for this `else` statement. _This `else` after return occurs several times here._